src/applications/vaos/utils/appointment.jsx

```diff
@@ -17,7 +17,15 @@ export function isCommunityCare(appt) {
 }
 
 export function isVideoVisit(appt) {
-  return !!appt.vvsAppointments;
+  return !!appt.vvsAppointments || appt.appointmentKind === 'MOBILE_GFE';
```

Comment:
Do we need to change `appointmentKind` here as well?

Reply:
Yes, good catch.  Thanks!